people.json

```diff
@@ -15491,10 +15491,10 @@
    {
       "name": "Matteo Bianchi",
       "bio": "<p>Matteo is a seasoned Software and Cloud Engineer but also a former startup CTO, DevRel and Solution Architect. Hacker and builder, advocating for Cloud Native technologies around the world as active member of the Cloud Native community. Matteo is a Kubernetes Release Team Member and active contributor who loves Platform Engineering. Co-organizer of KCD Netherlands and CNGC Netherlands meetups, helping out different KCDs and founding organizer of Cloud Native Days. @mbianchidev on (almost) every social media.</p>",
-      "company": "SyncTune",
-      "pronouns": "Other (please specify)",
+      "company": "GitHub",
+      "pronouns": "he/they",
       "location": "Amsterdam, Netherlands",
-      "twitter": "https://twitter.com/mbianchidev",
```

Comment:
Can you leave the "twitter" field in place and just set its value to ""? This will ensure we remove it from the CNCF website.